### PR TITLE
Formatted the code and added class level comments

### PR DESCRIPTION
**Pull request by vives**

### CODE REVIEW

**Comment by Niranjan-K** [comment]
@vives : the PR title and commit should be imperative. Also you can use git squash and have a single commit here.

File: component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java [hunk 1]
```diff
@@ -42,29 +43,35 @@
...
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 /**
- * Authenticator of Outlook
+ * Authenticator of Office365.
+ *
+ * @since 1.0.1
```
Comment (Niranjan-K) [inline component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java]: A new class was added in a patch release?

Comment (vives) [reply component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java]: Normally, if it is minor changes, we release tag like(1.0.2..).

Comment (Niranjan-K) [reply component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java]: Sorry but that is for the patch release. The version strategy is Major.Minor.Patch or are we following a different one for wso2-extensions repos?

Comment (vives) [reply component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java]: For wso2-extensions, once we fix or do improvement, we release extension, AFAIK, we follow Major.Minor version strategy. If it is minor improvement or fix, we release minor version and if API changes, we release major version.

**Comment by JKAUSHALYA** [comment]
Closing since not approved.

**Pull request closed without merging**